packages/ffmpeg/server/FFmpegExporterServer.ts

```diff
@@ -245,6 +254,34 @@ export class FFmpegExporterServer {
     });
   }
 
+  private async calculateAtempoFilters(playbackRate: number) {
+    const atempoFilters = [];
+
+    // Calculate how many times we need to 100x the speed
+    let rate = playbackRate;
+    while (rate > 100.0) {
+      atempoFilters.push('atempo=100.0');
+      rate /= 100.0;
+    }
+    // Add the last atempo filter with the remaining rate
+    if (rate > 1.0) {
+      atempoFilters.push(`atempo=${rate}`);
+    }
+
+    // Calculate how many times we need to halve the speed
+    rate = playbackRate;
+    while (rate < 0.5) {
+      atempoFilters.push('atempo=0.5');
```

Comment:
similar question, what happens if rate is 0? could this result in an infinite loop?

Reply:
great catch, that is exactly what happens - I've fixed this by just not including the audio when an asset's playbackRate (also volume) is 0.